personalized_nlp/datasets/emotions_perspective/emotions_perspectives.py

```diff
@@ -1,5 +1,5 @@
-from lib2to3.pytree import Base
-from tkinter import N
+# from lib2to3.pytree import Base
+# from tkinter import N
 from typing import List
```

Comment:
These commented imports can be deleted if unused.

Reply:
Alright, will do